packages/data-service/src/legacy/legacy-connection-model.ts

```diff
@@ -91,7 +91,7 @@ export interface LegacyConnectionModelProperties {
   kerberosServiceName?: string;
   kerberosPrincipal?: string;
   kerberosServiceRealm?: string;
-  kerberosCanonicalizeHostname?: string;
+  kerberosCanonicalizeHostname: boolean;
```

Comment:
Is this change from somewhere else? Seems unrelated

Reply:
Yeah I noticed this wrong type when was working on vscode: https://github.com/mongodb-js/vscode/blob/VSCODE-312-switch-to-new-data-service-and-model-v2/src/views/webview-app/connection-model/legacy-connection-model.ts#L40

Since it doesn't affect anything else decided to not create a separate ticket and pr for fixing it.